src/Console/Commands/EventStoreWorkerThread.php

```diff
@@ -35,7 +36,7 @@ public function __construct()
     public function handle(): void
     {
         if (!$this->option('stream')) {
-            $this->info("Stream option is required");
+            report("Stream option is required");
```

Comment:
This function takes an exception not a string? Are you sure it works?

Reply:
Good call! I assumed it takes string as well - updating